pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -221,3 +225,27 @@ func countEnabledFields(fields ...*string) int {
 	}
 	return count
 }
+
+func validateNodeGroupKubeletExtraConfig(kubeletConfig *NodeGroupKubeletConfig) error {
+	if kubeletConfig == nil {
+		return nil
+	}
+
+	var kubeletForbiddenFields = map[string]bool{
```

Comment:
nit: `map[string]struct{}` is usually used in places where the value doesn't matter

Reply:
thanks!